### PR TITLE
Add eng docker

### PR DESCRIPTION
**Pull request by LenaLenaPan**

### CODE REVIEW

File: docs/en_US/installation/docker.md [hunk 1]
```diff
@@ -1,83 +1,167 @@
...
+```bash
+docker pull emqx/nanomq:latest-slim
+```
 
-1. To get the Docker image, run:
+or Full version
 
 ```bash
-docker pull emqx/nanomq:lastest
+docker pull emqx/nanomq:latest-full
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:40]:
```suggestion
docker pull emqx/nanomq:0.18.2-full
```

File: docs/zh_CN/installation/docker.md [hunk 2]
```diff
@@ -64,14 +64,29 @@ NanoMQ 也支持通过配置文件或环境变量加载自定义配置。
 
    `docker cp nanomq.conf nanomq:/etc/nanomq.conf`
 
-以下为启用 TLS 连接的 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v014.md)：<!--这里的配置项需要更新吗？-->
+以下为 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v014.md)：
```
Comment (lee-emqx) [inline docs/zh_CN/installation/docker.md:67]: 改成最新的吧

File: docs/zh_CN/installation/docker.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ docker pull emqx/nanomq:latest-slim
 或完整版
 
 ```bash
-docker pull emqx/nanomq:0.14.1-full
+docker pull emqx/nanomq:latest-full
```
Comment (LenaLenaPan) [inline docs/zh_CN/installation/docker.md:40]:
```suggestion
docker pull emqx/nanomq:0.18.2-full
```

**Comment by JaylinYu** [comment]
nmq-wss is not supported yet. @RanMaoyi  remember to update this part of doc

File: docs/zh_CN/installation/docker.md [hunk 4]
```diff
@@ -64,14 +66,29 @@ NanoMQ 也支持通过配置文件或环境变量加载自定义配置。
 
    `docker cp nanomq.conf nanomq:/etc/nanomq.conf`
 
-以下为启用 TLS 连接的 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v014.md)：<!--这里的配置项需要更新吗？-->
+以下为 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v014.md)：
```
Comment (LenaLenaPan) [inline docs/zh_CN/installation/docker.md:69]:
```suggestion
以下为 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v018.md)：
```

File: docs/zh_CN/installation/docker.md (hunk 2 above)
Comment (LenaLenaPan) [reply docs/zh_CN/installation/docker.md:69]: done

File: docs/en_US/installation/docker.md [hunk 5]
```diff
@@ -1,83 +1,175 @@
...
+
+- Modify `/etc/nanomq.conf` in the Docker container
+
+- Copy the modified configuration file from your local machine to the Docker container's `/etc/nanomq.conf` path using the `docker cp` command:
+
+  ```bash
+  docker cp nanomq.conf nanomq:/etc/nanomq.conf
+  ```
+
+Here's an example configuration for enabling MQTT bridging with TLS connection. For more explanations on NanoMQ's configurations, please refer to the [Configuration Guide](../config-description/v014.md):
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:77]:
```suggestion
Here's an example configuration for enabling MQTT bridging with TLS connection. For more explanations on NanoMQ's configurations, please refer to the [Configuration Guide](../config-description/v018.md):
```

**Review by JaylinYu — APPROVED** [review]

File: docs/en_US/installation/docker.md [hunk 6]
```diff
@@ -1,83 +1,175 @@
...
-For more information about NanoMQ official docker image, see [Docker Hub - nanomq](https://hub.docker.com/r/emqx/nanomq)
+## Load Custom Configurations
+
+NanoMQ also allows loading custom configurations through a configuration file or environment variables.
+
+### Load through Configuration File
+
+If you wish to start NanoMQ via a configuration file:
+
+- Modify `/etc/nanomq.conf` in the Docker container, refer to [configruation description](../config-description/v018.md)
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:69]:
```suggestion
- Modify `/etc/nanomq.conf` in the Docker container, refer to [configruation description](../config-description/v019.md)
```

File: docs/en_US/installation/docker.md [hunk 7]
```diff
@@ -1,83 +1,175 @@
...
+
+- Copy the modified configuration file from your local machine to the Docker container's `/etc/nanomq.conf` path using the `docker cp` command:
+
+  ```bash
+  docker cp nanomq.conf nanomq:/etc/nanomq.conf
+  ```
+
+Here's an example configuration for enabling MQTT bridging with TLS connection. For more explanations on NanoMQ's configurations, please refer to the [Configuration Guide](../config-description/v018.md):
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:77]:
```suggestion
Here's an example configuration for enabling MQTT bridging with TLS connection. For more explanations on NanoMQ's configurations, please refer to the [Configuration Guide](../config-description/v019.md):
```

File: docs/zh_CN/installation/docker.md [hunk 8]
```diff
@@ -64,14 +66,29 @@ NanoMQ 也支持通过配置文件或环境变量加载自定义配置。
 
    `docker cp nanomq.conf nanomq:/etc/nanomq.conf`
 
-以下为启用 TLS 连接的 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v018.md)：<!--这里的配置项需要更新吗？-->
+以下为 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v018.md)：
```
Comment (LenaLenaPan) [inline docs/zh_CN/installation/docker.md:69]:
```suggestion
以下为 MQTT 桥接配置示例，更多关于 NanoMQ 的配置项说明，可参考 [配置说明](../config-description/v019.md)：
```

**Review by JaylinYu — APPROVED** [review]

File: docs/en_US/installation/docker.md [hunk 9]
```diff
@@ -1,83 +1,178 @@
...
+| NANOMQ_LOG_TO                   | String    | Log output types, separated by vertical line `|`<br />Values: file, console, syslog |
+| NANOMQ_LOG_LEVEL                | String    | Log level: trace, debug, info, warn, error, fatal            |
+| NANOMQ_LOG_DIR                  | String    | Path to store log files (effective when output is file)      |
+| NANOMQ_LOG_FILE                 | String    | Log file name (effective when output is a file)              |
+| NANOMQ_LOG_ROTATION_SIZE        | String    | Maximum occupied space per log file; <br />Unit: `KB | MB | GB`;<br />Default: `10MB` |
+| NANOMQ_LOG_ROTATION_COUNT       | Integer   | Maximum number of rotated log files;<br />Default: `5`       |
+| NANOMQ_CONF_PATH                | String    | NanoMQ configuration file path (default: `/etc/nanomq.conf`) |
+
+**Example: Specify the configuration file path through environment variables**
 - Modify `/etc/nanomq.conf ` on docker container, refer to [configruation description](../config-description/v019.md)
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:147]:
```suggestion

```

File: docs/en_US/installation/docker.md [hunk 10]
```diff
@@ -1,83 +1,178 @@
...
 - Copy your configuration file from local host to container path `/etc/nanomq.conf`:  `docker cp nanomq.conf nanomq:/etc/nanomq.conf`
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:148]:
```suggestion

```

File: docs/en_US/installation/docker.md [hunk 11]
```diff
@@ -1,83 +1,178 @@
...
 - Modify configuration parameters by environment variables, for example:
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:149]:
```suggestion

```

File: docs/en_US/installation/docker.md [hunk 12]
```diff
@@ -1,83 +1,175 @@
...
+
+**Example: Specify the configuration file path through environment variables**
+
+```bash
+docker run -d -p 1883:1883 -e NANOMQ_CONF_PATH="/usr/local/etc/nanomq.conf" \
+            [-v {LOCAL PATH}:{CONTAINER PATH}] \
+            --name nanomq emqx/nanomq:0.18.2-slim
+```
+
+## 性能调优
```
Comment (LenaLenaPan) [inline docs/en_US/installation/docker.md:154]:
```suggestion
## Performance Tunning
```

**Review by JaylinYu — APPROVED** [review]

**Pull request merged**